source/add-existing.txt

```diff
@@ -208,9 +205,9 @@ perform schema migration because MongoDB does not require
 a predefined schema to store the data.
 
 Migration tools are often specific to datasets.
-Even though {+odm+} supports a superset of ActiveRecord associations,
+Even though {+odm+} supports a superset of Active Record associations,
 the way that model references are stored in collections is different between
-{+odm+} and ActiveRecord.
+{+odm+} and Active Record.
```

Comment:
S: "{+odm+} and Active Record store model references in collections differently."